lib/Raven/Middleware/ExceptionInterfaceMiddleware.php

```diff
@@ -51,8 +51,9 @@ public function __construct(Client $client)
      */
     public function __invoke(Event $event, callable $next, ServerRequestInterface $request = null, $exception = null, array $payload = [])
     {
-        // Do not override the level if it was set explicitly by the user
-        if (!isset($payload['level']) && $exception instanceof \ErrorException) {
+        if (isset($payload['level'])) {
```

Comment:
Was this a bug? Can we put a test here?

Reply:
It wasn't a bug, I moved some additional code from the `Client::capture` method to the `ExceptionMiddleware` because I think it belongs there and the existing tests still pass